src/libcore/cmp.rs

```diff
@@ -512,16 +571,56 @@ impl PartialOrd for Ordering {
     }
 }
 
-/// Trait for values that can be compared for a sort-order.
+/// Trait for values that can be partially ordered.
+///
+/// This traits defines the operators `<`, `>`, `<=`, and `>=` which compare two values and return `bool`, 
+/// where `<` and `>` form [_strict_ partial ordering relations][wiki_porder].
```

Comment:
I would split this sentence. "[...] return `bool`. The operators `<` and `>` form [...]"

Reply:
I'd move the remark about strict ordering relations far down to all the corollaries.